common/lib/dependabot/shared_helpers.rb

```diff
@@ -90,6 +90,11 @@ def self.run_helper_subprocess(command:, function:, args:, env: nil,
       stdin_data = JSON.dump(function: function, args: args)
       cmd = allow_unsafe_shell_command ? command : escape_command(command)
       env_cmd = [env, cmd].compact
+      if ENV["DEBUG_FUNCTION"] == function
```

Comment:
@jurre thoughts on this? A bit crude but found it helpful

Reply:
I've done something similar to test the native helpers, I think it's OK for now but ideally would love to have a better story for working on those helpers